uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/models/HabitListTest.kt

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (C) 2016-2021 Álinson Santos Xavier <git@axavier.org>
+ *
+ * This file is part of Loop Habit Tracker.
+ *
+ * Loop Habit Tracker is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * Loop Habit Tracker is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isoron.uhabits.core.models
+
+import junit.framework.TestCase
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.isoron.uhabits.core.BaseUnitTest
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import java.io.IOException
+import java.io.StringWriter
+import java.util.ArrayList
+
+class HabitListTest : BaseUnitTest() {
+    @get:Rule
+    var thrown = ExpectedException.none()!!
+    private var habitsArray: ArrayList<Habit>? = null
+    private var activeHabits: HabitList? = null
+    private var reminderHabits: HabitList? = null
+    @Throws(Exception::class)
+    override fun setUp() {
+        super.setUp()
+        habitsArray = ArrayList()
+        for (i in 0..9) {
+            val habit = fixtures!!.createEmptyHabit()
+            habitList!!.add(habit)
+            habitsArray!!.add(habit)
+            if (i % 3 == 0) habit.reminder = Reminder(8, 30, WeekdayList.EVERY_DAY)
+        }
+        habitsArray!![0].isArchived = true
+        habitsArray!![1].isArchived = true
+        habitsArray!![4].isArchived = true
+        habitsArray!![7].isArchived = true
+        activeHabits = habitList!!.getFiltered(HabitMatcherBuilder().build())
+        reminderHabits = habitList!!.getFiltered(
+            HabitMatcherBuilder()
+                .setArchivedAllowed(true)
+                .setReminderRequired(true)
+                .build()
+        )
+    }
+
+    @Test
+    fun testSize() {
+        MatcherAssert.assertThat(habitList!!.size(), CoreMatchers.equalTo(10))
+        MatcherAssert.assertThat(activeHabits!!.size(), CoreMatchers.equalTo(6))
+        MatcherAssert.assertThat(reminderHabits!!.size(), CoreMatchers.equalTo(4))
```

Comment:
Please use static imports in `assertThat` and `equalTo`. Same for the other usages in this file.